src/Dapr.Workflow/WorkflowServiceCollectionExtensions.cs

```diff
@@ -153,31 +149,33 @@ static bool TryGetGrpcAddress(out string address)
             return false;
         }
 
-        static GrpcChannel CreateChannel(string address, HttpClient client)
+        static GrpcChannel CreateChannel(string address, HttpClient client, GrpcChannelOptions? grpcChannelOptions = null)
```

Comment:
This is also used by workflow clients, right?  Would they not have similar needs to customize gRPC options as well?

Reply:
Hey @philliphoff this has been addressed couple of weeks ago, could you take a look? Many thanks!